modular_citadel/code/modules/client/loadout/backpack.dm

```diff
@@ -41,12 +41,10 @@
 /datum/gear/backpack/tennis/tri
 	name = "Tri-color Polychromic Tennis Ball"
 	path = /obj/item/toy/fluff/tennis_poly/tri
-	cost = 3
```

Comment:
Would it be too much to ask to instead of editing the nonmodular loadout files you just updated the prices in our modular code?